src/clients/nick-client.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const Client = require('./base-browser-client')
+
+class NickClient {
```

Comment:
First question, these clients are only intended for one resource. That's why there is a `UserClient` and a `ContactClient` but not a `DoorkeeperClient.`

Are you sure you want to name it `NickClient` and not `NickObjectClient`, because you will need more clients for nick later?